### PR TITLE
Optimize build.html on the /output route for mobiles

### LINKED ISSUE #301: Optimize build.html page for mobiles that's displayed in /output route
**I'm submitting a ...**  (check one with "x")
- [x] bug report
- [ ] feature request
- [ ] support request => Please do not submit support requests here, instead ask your query in out Gitter channel at https://gitter.im/fossasia/meilix

**Actual Behaviour**
image
As you can see, everything's overlapping.

**Expected Behaviour**
Would be really nice if we optimize it for phones.

**Other information:**

Minor mods to the html and css files

**Would you like to work on the issue?**
Yes

### PR DESCRIPTION
**Pull request by rahulkumaran**
#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] There is a corresponding issue for this pull request.
- [x] My branch is up-to-date with the Upstream `master` branch.
- [ ] The unit tests pass locally with my changes (provide a screenshot or link for test) 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

#### Short description of what this resolves:
This PR essentially makes sure that the build.html file has been optimised when viewed on mobile devices too.
<b>Before Optimization</b>
image

<b>After Optimization</b>
image

#### Changes proposed in this pull request:
- Add link for email.css stylesheet in layout.html
- Optimise build.html for mobile devies 

Fixes #301

### CODE REVIEW

**Comment by pranav1698** [comment]
Nice work @rahulkumaran, I think we should also decrease the size of those email-provider images as we switch to mobile size

**Comment by rahulkumaran** [comment]
Thanks @pranav1698 ! 
Yeah, I was thinking of the same. But let's see what the others have to say! Open to mods as per the reviews haha :smile:

**Comment by xeon-zolt** [comment]
@rahulkumaran can we use better design doesn't look good

File: templates/build.html [hunk 1]
```diff
@@ -43,5 +41,5 @@
 
 
 </div>
-
+<br/><br/><br/>
```
Comment (r0mflip) [inline templates/build.html]: Use margins though CSS, don't use `<br>`

**Comment by rahulkumaran** [comment]
@r0mflip @tabesin, sure! Will update it by Friday. Currently have my exams going on

**Comment by r0mflip** [comment]
> Currently have my exams going on

@rahulkumaran :rofl: me too

File: templates/build.html [hunk 2]
```diff
@@ -11,24 +11,24 @@
 </main>
 <div class="container">
 
-  <div class="alert alert-info" >
+  <div class="alert alert-info" style="margin-botton:30px;">
```
Comment (r0mflip) [inline templates/build.html]: Avoid inline CSS

File: templates/build.html [hunk 3]
```diff
@@ -11,24 +11,24 @@
...
     <strong>Check your Email Now</strong>
     <div class="container-fluid">
       <div class="row">
 
         <a class="col-sm-3" target="_blank" href="https://www.gmail.com">
-          <img id='mail'  src="{{ url_for('static',filename='gmail.png') }}">
+          <img id='mail' width="130" style="margin-top:15px;"src="{{ url_for('static',filename='gmail.png') }}">
```
Comment (r0mflip) [inline templates/build.html]: Avoid inline CSS

Comment (rahulkumaran) [reply templates/build.html]: > > Currently have my exams going on
> 
> @rahulkumaran rofl me too

Haha, finally done with them! xD

Comment (rahulkumaran) [reply templates/build.html]: @r0mflip Oops! My bad! Will add them to the necessary file!

**Pull request closed without merging**